lib/coinbase/user.rb

```diff
@@ -20,21 +20,11 @@ def id
     end
 
     # Creates a new Wallet belonging to the User.
+    # @param network_id [String] (Optional) the ID of the blockchain network. Defaults to 'base-sepolia'.
+    # @param server_signer [Boolean] (Optional) whether Wallet should use project's server signer. Defaults to false.
```

Comment:
Should we document saying this is a no-op right now?

Reply:
Moving to a feature branch!